src/utilities/getVTokenByAddress.ts

```diff
@@ -2,8 +2,8 @@ import { VToken } from 'types';
 
 import { VBEP_TOKENS } from 'constants/tokens';
 
-const getVTokenByAddress = (address: string) =>
-  address.toLowerCase() in VBEP_TOKENS
+const getVTokenByAddress = (address: string | null) =>
```

Comment:
In this case we could just make address optional, so that undefined could be passed to the function as well

Reply:
Updated